packages/astro/src/core/errors/errors.ts

```diff
@@ -27,7 +27,7 @@ type ErrorTypes =
 	| 'AggregateError';
 
 export function isAstroError(e: unknown): e is AstroError {
-	return e instanceof AstroError;
+	return e instanceof AstroError || AstroError.is(e);
```

Comment:
Why the change?

Reply:
When the AstroError is thrown inside the content config it's not recognising it as an instance of the AstroError class, so isn't giving the nicely-formatted error messages